ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -200,6 +244,35 @@ device_classes: !ZenPackSpec
                                 dpName: volume_free
                                 lineType: AREA
                                 rpn: "size,-,size,/,-100,*"
+                    Physical Volume IO Stats:
+                        units: per sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            RRQM:
+                                dpName: diskstats_numReadsMerged
+                                lineType: AREA
+                                rpn: "${here/sysUpTime},/,100,*"
```

Comment:
You're already capturing these values as DERIVE, which means they'll already be shown as value/second. I don't see why you'd then want that number as a percentage of the system's uptime.

Even if you did want that for some reason, you can't really use a model value like here/sysUpTime like that because it'll only be interpreted when the monitoring configuration is being rebuilt. So the sysUpTime portion of the formula will remain static for many hours.